src/index.js

```diff
@@ -64,6 +84,8 @@ function getTriggeringProps<PropsInput: {}>(
   return propNames.map(name => props[name])
 }
 
+const hasOwn = Object.prototype.hasOwnProperty
```

Comment:
Any particular reason you assigned this to a variable instead of using it directly?

Reply:
not really, I don't remember why i did that ;)